### PR TITLE
Remove flaky case from 02956_rocksdb_bulk_sink

### PR DESCRIPTION
**Pull request by vdimir**
### Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 61845158171d00841dbbe91aeafefdee7b247198 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/66107/61845158171d00841dbbe91aeafefdee7b247198/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Review by yariks5s — APPROVED** [review]

File: tests/queries/0_stateless/02956_rocksdb_bulk_sink.sh [hunk 1]
```diff
@@ -25,13 +25,6 @@ ${CLICKHOUSE_CLIENT} --query "INSERT INTO rocksdb_worm SELECT number, number+1 F
 ${CLICKHOUSE_CLIENT} --query "SELECT sum(value) FROM system.rocksdb WHERE database = currentDatabase() AND table = 'rocksdb_worm' AND name = 'no.file.opens';" # should be 1
 ${CLICKHOUSE_CLIENT} --query "SELECT count() FROM rocksdb_worm;"
 
-# Testing insert with multiple sinks and fixed block size
-${CLICKHOUSE_CLIENT} --query "TRUNCATE TABLE rocksdb_worm;"
-# Must set both max_threads and max_insert_threads to 2 to make sure there is only two sinks
-${CLICKHOUSE_CLIENT} --query "INSERT INTO rocksdb_worm SELECT number, number+1 FROM numbers_mt(1000000) SETTINGS max_threads = 2, max_insert_threads = 2, max_block_size = 10000, min_insert_block_size_rows = 0, min_insert_block_size_bytes = 0, insert_deduplication_token = '', optimize_trivial_insert_select = 1;"
-${CLICKHOUSE_CLIENT} --query "SELECT sum(value) FROM system.rocksdb WHERE database = currentDatabase() AND table = 'rocksdb_worm' AND name = 'no.file.opens';" # should be 2 because default bulk sink size is ~1M rows / SST file
```
Comment (azat) [inline tests/queries/0_stateless/02956_rocksdb_bulk_sink.sh:32]: Maybe remove only the check for `no.file.opens`?

**Comment by vdimir** [comment]
> Builds Pending — 13 of 21 builds are missing. 18/31 artifact groups are OK [Details](https://s3.amazonaws.com/clickhouse-test-reports/66107/859c63298ef044d92c585921579f90dd6e56deda/builds/report.html)

**Comment by alexey-milovidov** [comment]
@vdimir tests have failed.

**Comment by alexey-milovidov** [comment]
Builds didn't even finish.

**Pull request merged**